W-10796831: Update docs (#111)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-dates-functions-time.adoc

[%header, cols="1,3"]
|===
| Name | Description
| `parts` | `Number` values for `hour`, `minutes`, and `seconds` fields, and a
            `TimeZone` value for the `timezone` field. Valid values are 0

>             through 23 for the `hour`, 0 through 59 for `minutes`, and 0 through 59 (not inclusive; it can accept 59.99, for example) for `seconds` fields. The `timezone` must be a valid `TimeZone` value,

Reviewer: TODO: CHECK "not inclusive" here, too. # 4 instance

>             through 23 for the `hour`, 0 through 59 for `minutes`, and 0 through 59 (not inclusive; it can accept 59.99, for example) for `seconds` fields. The `timezone` must be a valid `TimeZone` value,

Reviewer: ```suggestion
            through 23 for the `hour`, 0 through 59 for `minutes`, and 0 through 59 (including decimals, such as 59.99) for `seconds` fields. The `timezone` must be a valid `TimeZone` value,
```

Reviewer: fixed in source, master

>             such as `&#124;-03:00&#124;` You can specify the name-value pairs in any

Reviewer: TODO: CHECK that HTML code is converted to pipe here; prob does.

Reviewer: it gets converted when in backticks

output application/json
---
{

>   newTime: time({ hour: 12, minutes: 30, seconds: 40 , timeZone: &#124;-03:00&#124;})

Reviewer: TODO: CHECK if the HTML code gets transformed to a pipe in this example, prob not.

>   newTime: time({ hour: 12, minutes: 30, seconds: 40 , timeZone: &#124;-03:00&#124;})

Reviewer: ```suggestion
  newTime: time({ hour: 12, minutes: 30, seconds: 40 , timeZone: |-03:00|})
```
